stripe_billing_dashboard/tasks/list_stripe_customers.js

```diff
@@ -0,0 +1,21 @@
+// TODO: Set your secret key here. This is a test Stripe API key.
+// See your Stripe API keys here: https://dashboard.stripe.com/apikeys
+const stripe = require('stripe')('sk_test_4eC39HqLyjWDarjtT1zdp7dc');
```

Comment:
minor but I'd like to separate out the import

```suggestion
import Stripe from "stripe";
// TODO: Set your secret key here. This is a test Stripe API key.
// See your Stripe API keys here: https://dashboard.stripe.com/apikeys
const stripe = Stripe('sk_test_4eC39HqLyjWDarjtT1zdp7dc');
```